quizer_game/templates/quizer_game/edit-question-set.html

```diff
@@ -2,9 +2,263 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
```

Comment:
If this page is opened, is the page title appeared?

Reply:
> 
> 
> If this page is opened, is the page title appeared?

I already fix it.